marimo/_utils/formatter.py

```diff
@@ -42,8 +45,17 @@ def format(self, codes: CellCodes) -> CellCodes:
 
 class RuffFormatter(Formatter):
     def format(self, codes: CellCodes) -> CellCodes:
+        # first look for ruff next to sys.executable
+        ruff_path = Path(sys.prefix) / "ruff"
```

Comment:
Shouldn't this be `Path(sys.prefix) / "bin" / "ruff"`? And why is this preferable over ="python -m ruff"`?

Reply:
Okay I understand why not the latter. But how about the former, isn't `bin` necessary?

Reply:
I think that leaves the same ambiguity as to where python is running from as I'm trying to address for ruff.
However, I incorrectly assumed that `Path(sys.prefix) = Path(sys.executable).parent` - you're right that "bin" is missing. My bad.  However, on Windows it's in `Scripts`.

So your suggestion is really optimal and simple here.